src/components/BlogPage/BlogBack.tsx

```diff
@@ -0,0 +1,13 @@
+import { Button } from "@mui/material";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+
+const BlogBack = () => {
+  return (
+    <Button sx={{ my: 2 }} href="/blog">
+      <ArrowBackIcon />
```

Comment:
Use the button prop 'startIcon'

`<Button startIcon={<ArrowBack />}>
Back
</Button>`

also does using the href like that cause the page to reload when navigating? if so, could you use the router from next/navigation and `router.push('/blog')`